package.json

```diff
@@ -8,7 +8,6 @@
     "@fortawesome/react-fontawesome": "^0.1.4",
     "@storybook/addon-storyshots": "^5.2.5",
     "@zooniverse/grommet-theme": "^2.1.0",
-    "@zooniverse/panoptes-js": "^0.1.1",
```

Comment:
I'm not super familiar with `panoptes-js`, was there anything unique it was doing, that PJC doesn't do?

Reply:
Related updated requests look good, nevermind.